src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/GetAzureSqlDatabaseBackupLongTermRetentionPolicy.cs

```diff
@@ -21,23 +21,32 @@
 namespace Microsoft.Azure.Commands.Sql.Backup.Cmdlet
 {
     [Cmdlet(VerbsCommon.Get, "AzureRmSqlDatabaseBackupLongTermRetentionPolicy", SupportsShouldProcess = true)]
+    [Alias("Get-AzureRmSqlDatabaseLongTermRetentionPolicy")]
     public class GetAzureSqlDatabaseBackupLongTermRetentionPolicy : AzureSqlDatabaseBackupLongTermRetentionPolicyCmdletBase
     {
+        /// <summary>
+        /// Gets or sets whether or not to use the Long Term Retention Vaults.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
Remove valuefrompipelinebypropertyname and position

Reply:
Done. Also changed to current instead of legacy so we do not break existing workflows until LTR V1 is deprecated.